gp-populate-anything/gppa-aggregate-functions.php

```diff
@@ -56,12 +56,17 @@ public function init() {
 
 	}
 
-	public function enable_query_all_value_objects( $query_all_value_objects, $field, $field_values, $object_type_instance, $filter_groups, $primary_property, $templates ) {
-		return $this->matches_any_merge_tag( rgar( $templates, 'value' ) );
+	public function enable_query_all_value_objects( $should_query_all, $field, $field_values, $object_type_instance, $filter_groups, $primary_property, $templates ) {
+		return $should_query_all || $this->matches_any_merge_tag( rgar( $templates, 'value' ) );
 	}
 
+	// The count function normally has "query all value objects" enabled, but this function overrides that.
+	// We need to specify that count has to have "query all value objects" enabled to avoid breaking it.
```

Comment:
This comment is a bit too ambiguous to be useful for future us. My recommendation would be to ditch it.